packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1159,6 +1159,10 @@ abstract class TextInputClient {
 
   /// Requests that the client remove the text placeholder.
   void removeTextPlaceholder() {}
+
+  /// Performs macOS specific selector from the `NSStandardKeyBindingResponding`
+  /// protocol or user specified selector from `DefaultKeyBinding.Dict`.
+  void performSelector(String selectorName) {}
```

Comment:
Could this be more generic than text input?  Maybe it should have its own class instead of being a part of TextInputClient?

Reply:
The selector is sent from TextInputContext, which is only used through the TextInputClient.